src/Illuminate/Database/Query/JoinClause.php

```diff
@@ -46,12 +46,19 @@ public function __construct($type, $table)
     }
 
     /**
-     * Add an "on" clause to the join.
-     *
-     * @param  string  $first
-     * @param  string  $operator
-     * @param  string  $second
-     * @param  string  $boolean
+     * Add an "on" clause to the join. On clauses can be chained, e.g.
+     * 
+     * $join->on('contacts.user_id', '=', 'users.id')
+     *      ->on('contacts.info_id', '=', 'info.id')
+     * 
+     * will produce the following SQL:
+     * 
+     * on `contacts`.`user_id` = `users`.`id`  and `contacts`.`info_id` = `info`.`id`
+     * 
+     * @param  string  $first The first argument (column) to compare with
```

Comment:
We never document params like this, and even if we did, you'd need to put 2 spaces before the docs, not 1.